film_loader.py

```diff
@@ -0,0 +1,123 @@
+        (
+            select GROUP_CONCAT(actor_id) from
+            (
+                select actor_id
+                from movie_actors
+                where movie_id = movies.id
+            )
+        ),
+
+        max(writer, writers)
+        from movies
+    """)
+
+    raw_data = cursor.fetchall()
+
+    # cursor.execute('pragma table_info(movies)')
+    # pprint(cursor.fetchall())
+
+    # РќСѓР¶РЅС‹ РґР»СЏ СЃРѕРѕС‚РІРµС‚СЃРІРёСЏ РёРґРµРЅС‚РёС„РёРєР°С‚РѕСЂР° Рё С‡РµР»РѕРІРµРєРѕС‡РёС‚Р°РµРјРѕРіРѕ РЅР°Р·РІР°РЅРёСЏ
+    actors = {row[0]: row[1] for row in cursor.execute('select * from actors where name != "N/A"')}
+    writers = {row[0]: row[1] for row in cursor.execute('select * from writers where name != "N/A"')}
+
+    return actors, writers, raw_data
+
+
+def transform(__actors, __writers, __raw_data):
+    """
+
+    :param __actors:
+    :param __writers:
+    :param __raw_data:
+    :return:
+    """
+    documents_list = []
+    for movie_info in __raw_data:
+        # Р Р°Р·С‹РјРµРЅРѕРІР°РЅРёРµ СЃРїРёСЃРєР°
+        movie_id, imdb_rating, genre, title, description, director, raw_actors, raw_writers = movie_info
+
+        if raw_writers[0] == '[':
+            parsed = json.loads(raw_writers)
+            new_writers = ','.join([writer_row['id'] for writer_row in parsed])
+        else:
+            new_writers = raw_writers
+
+        writers_list = [(writer_id, __writers.get(writer_id)) for writer_id in new_writers.split(',')]
+        actors_list = [(actor_id, __actors.get(int(actor_id))) for actor_id in raw_actors.split(',')]
+
+        document = {
+            "_index": "movies",
+            "_id": movie_id,
+            "id": movie_id,
+            "imdb_rating": imdb_rating,
+            "genre": genre.split(', '),
+            "title": title,
+            "description": description,
+            "director": director,
+            "actors": [
+                {
+                    "id": actor[0],
+                    "name": actor[1]
+                }
+                for actor in set(actors_list) if actor[1]
+            ],
+            "writers": [
+                {
+                    "id": writer[0],
+                    "name": writer[1]
+                }
+                for writer in set(writers_list) if writer[1]
+            ]
+        }
+
+        for key in document.keys():
+            if document[key] == 'N/A':
+                # print('hehe')
+                document[key] = None
+
+        document['actors_names'] = ", ".join([actor["name"] for actor in document['actors'] if actor]) or None
+        document['writers_names'] = ", ".join([writer["name"] for writer in document['writers'] if writer]) or None
+
+        import pprint
+        pprint.pprint(document)
+
+        documents_list.append(document)
+
+    return documents_list
+
+
+def load(acts):
+    """
+
+    :param acts:
+    :return:
+    """
+    es = Elasticsearch([{'host': '192.168.1.252', 'port': 9200}])
```

Comment:
Секреты правильно получать через переменные окружения. Было бы правильнее(и безопаснее) сделать так:
```suggestion
    es = Elasticsearch([{'host': os.environ.get("ELASTICSEARCH_HOST", "localhost"), 'port': os.environ.get("ELASTICSEARCH_PORT", 9200)}])
```